pkg/cmd/server/api/helpers.go

```diff
@@ -241,6 +241,14 @@ func GetMasterFileReferences(config *MasterConfig) []*string {
 
 		refs = append(refs, &config.KubernetesMasterConfig.ProxyClientInfo.CertFile)
 		refs = append(refs, &config.KubernetesMasterConfig.ProxyClientInfo.KeyFile)
+
+		for _, key := range []string{"cluster-signing-cert-file", "cluster-signing-key-file"} {
```

Comment:
I was going to do it for all flags that are file paths, but it's a super long list.  No disagreement that we *should* do this though?